massa-graph/src/block_graph.rs

```diff
@@ -1200,18 +1237,23 @@ impl BlockGraph {
         // for each active block
         for block_id in self.active_index.iter() {
             if let Some(BlockStatus::Active(active_block)) = self.block_statuses.get(block_id) {
+                // TODO: clone operations in one go as in `get_operations_involving_address`.
```

Comment:
Maybe we should directly query storage for the not graph related api queries. Maybe more in a follow up

Reply:
For now I don't think so, because it's the graph that has the logic about whether a block should be returned or not, for example here we only use blocks that are `BlockStatus::Active`, and storage does not, and I think should not, know about this.